user_guide/intro_nav_popup.rst

```diff
@@ -0,0 +1,50 @@
+.. _user-guide-intro-popups:
+
+Flash Notifications and Dialog Windows
+======================================
+
+In the course of the system usage, there sometimes appears the need to inform the user about some events in the system,
+for example some operation has been executed or has failed, or you need to be reminded about a task or event.
+
+Success flash messages are displayed against a green background:
+
+|
+
+.. image:: ./img/navigation/popup/flash_success.png
+
+|
+
+If something has gone wrong, the flash message is displayed against a red background, to attract  your attention and 
+make sure you have noticed it. 
+
+|
+
+.. image:: ./img/navigation/popup/flash_error.png
+
+|
+
+Other flash messages have a yellow background. 
+
+|
+
+.. image:: ./img/navigation/popup/flash_task.png
+
+|
+
+Sometimes, the flash messages can contain some links to other pages, such as the task reminder above, or this 
+synchronization message:
```

Comment:
Edit: Sometimes, a flash message can contain links to other pages, such as the task reminder above, or this synchronization message: